python/clicastInterface.py

```diff
@@ -201,9 +201,11 @@ def rebuildEnvironment(enviroPath, jsonOptions):
         rebuildEnvironmentUsingClicastReBuild(enviroPath)
 
 
+codeTestCompileErrorCode=98
```

Comment:
Actions failed because of a `black` diff:

```patch
diff --git a/python/clicastInterface.py b/python/clicastInterface.py
index 8cc80e9..4d9f10b 100644
--- a/python/clicastInterface.py
+++ b/python/clicastInterface.py
@@ -201,7 +201,9 @@ def rebuildEnvironment(enviroPath, jsonOptions):
         rebuildEnvironmentUsingClicastReBuild(enviroPath)


-codeTestCompileErrorCode=98
+codeTestCompileErrorCode = 98
+
+
 def executeTest(testIDObject):
     # since we are doing a direct call to clicast, we need to quote the parameters
     # separate variable because in the future there will be additional parameters
```